README.md

```diff
@@ -81,6 +81,27 @@ puts response.status_code
 puts response.body
 puts response.headers
 ```
+## Manage Account Settings
```

Comment:
This should go in the README in the helper file, see the mail helper for an example: https://github.com/sendgrid/sendgrid-ruby/tree/master/lib/sendgrid/helpers/mail. Then you can link to it from here.